asgs_main.sh

```diff
@@ -3171,7 +3171,7 @@ while [ true ]; do
       CONTROLOPTIONS="$CONTROLOPTIONS --periodicflux $PERIODICFLUX"  # for specifying constant periodic flux
       RMQMessage "INFO" "$CURRENT_EVENT" "$THIS>$ENSTORM" "$CURRENT_STATE" "Generating ADCIRC Control File (fort.15) for $ENSTORM."
       logMessage "$ENSTORM: $THIS: Generating ADCIRC Control File (fort.15) for $ENSTORM with the following options: $CONTROLOPTIONS."
-
+      CONTROLOPTIONS="$CONTROLOPTIONS --nscreen $NSCREEN"
```

Comment:
I had a question - will the default for `$NSCREEN` always be available via `io_defaults.sh` or should this default value of `-1000` actually reside in `asgs_main.sh`?

Reply:
Ok I have added this to `variables_init()`